package.json

```diff
@@ -54,11 +55,14 @@
     "eslint-plugin-vue": "^8.7.1",
     "esno": "^0.16.3",
     "ethers": "^5.7.2",
-    "express": "^4.18.1",
+    "express": "^4.18.2",
```

Comment:
I don't think these dependencies belong here. Put them in their respective directories/packages using `npm i <external-package> --workspace @casimir/<our-package>`